CDP4CrossViewEditor/WorkbookOperator.cs

```diff
@@ -42,9 +42,31 @@ namespace CDP4CrossViewEditor
 
     using NetOffice.ExcelApi;
     using NetOffice.ExcelApi.Enums;
+    using NetOffice.Exceptions;
 
     using NLog;
 
+    /// <summary>
+    /// Workbook metadata that will be preserved between two running sessions
+    /// </summary>
+    public struct WorkbookMetadata
+    {
+        /// <summary>
+        /// Element definitions user selection
+        /// </summary>
+        public IEnumerable<ElementDefinition> ElementDefinitions;
+
+        /// <summary>
+        /// Parameter types user selection
+        /// </summary>
+        public IEnumerable<ParameterType> ParameterTypes;
+
+        /// <summary>
+        /// Parameter sheet values changed by the user
+        /// </summary>
+        public Dictionary<string, string> ParameterValues;
```

Comment:
`CrossviewWorkbookdata:141` should be consistent with this.

Maybe rename both to `ManuallySavedParameterValues`.